### ISSUE TITLE
Combined Loss Functions

### ISSUE THREAD

**Issue by andrewdalpino** [RubixML/ML#439]
Currently, you can specify one Cost Function for each model using the neural network subsystem under the hood. This task is to implement a combined Cost Function that takes any number of base Cost Functions and combines their loss (Either equal, weighted, or perhaps based on homoscadastic uncertainty).

**Comment by andrewdalpino** [comment]
Nah
